packages/api-explorer/src/PathUrl.jsx

```diff
@@ -9,11 +9,20 @@ const { Operation } = Oas;
 const extensions = require('@readme/oas-extensions');
 
 function splitPath(path) {
+  const keys = [];
```

Comment:
You could use a Set for this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Looks like we can use it in ie11: https://kangax.github.io/compat-table/es6/#test-Set

Reply:
Ignore this. I misunderstood.